packages/dapp/src/components/Navbar/Menu/MainMenu/MainMenu.tsx

```diff
@@ -53,40 +44,39 @@ export const MainMenu = ({ handleClose }: MainMenuProps) => {
             label={el.label}
             prefixIcon={el.prefixIcon}
             triggerSubMenu={el.triggerSubMenu}
-            showButton={el.showButton}
+            showButton={el.showButton || false}
```

Comment:
Is there another solution? 🙂

Reply:
What do you think about replacing it with this:
`showButton={el.showButton ?? false}`

Reply:
It´s complaining otherwise if I just pass it like so:
`showButton={el.showButton}
`
![image](https://github.com/lifinance/jumper.exchange/assets/43956540/aafdf052-7a56-4eff-a1e9-8ed442a48472)

Reply:
Or shall I define the type to be `boolean | undefined`  to make it cleaner?